install_config/router/default_haproxy_router.adoc

```diff
@@ -336,6 +336,81 @@ $ openssl ciphers
+Mutual TLS authentication is implemented on client certificates and can be
+controlled based on the certifying authorities (CAs) issuing the certificates,
+the certificate revocation list and/or any certificate subject filters.
+Use the mutual tls config options `--mutual-tls-auth`, `--mutual-tls-auth-ca`,
+`--mutual-tls-auth-crl` and `--mutual-tls-auth-filter` when creating a router:
+
+----
+$ oc adm router --mutual-tls-auth=required  \
+                --mutual-tls-auth-ca=/local/path/to/cacerts.pem   ....
+----
+
+The `--mutual-tls-auth` values are: `required`, `optional`, or `none`, with
+`none` as the default.
+The `--mutual-tls-auth-ca` value optionally specifies a file containing one or
+more CA certificates. These CA certificate[s] are used by the router to verify
+a client's certificate.
+
+The `--mutual-tls-auth-crl` can be optionally used specify the certificate
+revocation list to handle cases where certificates (issued by valid certifying
+authorities) have been revoked.
+
+----
+$ oc adm router --mutual-tls-auth=required  \
+     --mutual-tls-auth-ca=/local/path/to/cacerts.pem  \
+     --mutual-tls-auth-filter='^/CN=my.org/ST=CA/C=US/O=Security/OU=OSE$'  \
+     ....
+----
+
+The optional `--mutual-tls-auth-filter` value can be used for fine grain
+access control based on the certificate subject. The value is a regular
+expression which is to used to match up the certificate's subject.
+
+[NOTE]
+====
+The mutual TLS authentication filter example above shows you a restrictive
+regular expression (regex) - anchored with `^` and `$` - that `exactly` matches
+a certificate subject.  If you decide to use a less restrictive regular
+expression, please be aware that this can potentially match certificates
+issued by any certifying authorities (CAs) you have deemed to be valid.
+It is recommended to also use the `--mutual-tls-auth-ca` option so that you
+have finer control over the issued certificates.
+====
+
+Using `--mutual-tls-auth=required` ensures that you *_only_* allow authenticated
+clients access to the backend resources. This means that the client is always
+*_required_* to provide authentication information (aka a client certificate).
+To make the mutual TLS authentication optional, use `--mutual-tls-auth=optional`
+(or use `none` to disable it - this is the default).
+Note here that `optional` means that you do *_not_* require a client to present
+any authentication information and if the client provides any authentication
+information, that is just passed on to the backend in the `X-SSL*` HTTP headers.
+
+----
+$ oc adm router --mutual-tls-auth=optional  \
+     --mutual-tls-auth-ca=/local/path/to/cacerts.pem  \
+     ....
+----
+
+When mutual TLS authentication support is enabled (either using the `required`
+or `optional` value for the `--mutual-tls-auth` flag), the client authentication
+information is passed to the backend in the form of `X-SSL*` HTTP headers.
+
+Examples of the `X-SSL*` HTTP headers::
```

Comment:
s/Examples of the `X-SSL*` HTTP headers::/*Examples of the `X-SSL*` HTTP headers*